docker/dev/Dockerfile.django

```diff
@@ -0,0 +1,32 @@
+# We use the official Python images. This may need to get locked in, or back to Ubuntu at some point
+FROM python:3
+
+RUN apt-get update
+RUN apt-get install -y lib32z1-dev lib32z1 libpq-dev
```

Comment:
I'm kind of surprised `libxml2-dev libxslt-dev` are not needed. Those are direct dependencies for `lxml` to be built successfully. Maybe the Python3 image has them already?